common-pipeline/operators/conditions-v3-schema.json

```diff
@@ -3,7 +3,7 @@
   "id": "http://api.dataplatform.ibm.com/schemas/common-pipeline/operators/conditions-v3-schema.json#",
   "title": "Watson Data Platform Validation Schema",
   "description": "User Interface Condition schema for the Watson Data Platform",
-  "type": "object",
+  "type": "array",
```

Comment:
This seems wrong.  This isn't an array.  Can you explain why this should be changed?

Reply:
This `type` defines the type of the overall `conditions` which is an array of objects. Passing in the textfield_paramDef.json example from test harness into the validator returned 3 errors, one is for the conditions. (I just updated the issue description with the errors the validator returned)

<img width="643" alt="image" src="https://user-images.githubusercontent.com/1628100/104478451-2a0e9500-5577-11eb-87a2-539968de0ff4.png">
<img width="219" alt="image" src="https://user-images.githubusercontent.com/1628100/104478724-7ce84c80-5577-11eb-98cc-0d380e38c42a.png">

Reply:
That is defined in paramDef schema.  Doesn't that schema need to get updated instead to have array and items?

Reply:
I updated the paramDef schema instead to reference the exact definition